docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -2075,6 +2075,108 @@ Converts a string to its lowercase equivalent.
 --------------------------------------------------
 // NOTCONSOLE
 
+[[pipeline-processor]]
+=== Pipeline Processor
+Executes another pipeline.
+
+[[pipeline-options]]
+.Pipeline Options
+[options="header"]
+|======
+| Name             | Required  | Default  | Description
+| `pipeline`       | yes       | -        | The name of the pipeline to execute
+|======
+
+[source,js]
+--------------------------------------------------
+{
+  "pipeline": {
+    "pipeline": "inner-pipeline"
```

Comment:
Looking at actual requests, this looks quite confusing having "pipeline" inside "pipeline" (we used to have similar for `script` with the script query containing `script`). I wonder if we could quickly rename the parameter to `name` or something similar before 6.5 branch is cut?

Reply:
@rjernst sure  that makes sense. Should I just do it in this PR or do you want  a separate one?

Reply:
I would do a quick separate one, since this one is marked as docs.

Reply:
ok incoming :)

Reply:
Done in https://github.com/elastic/elasticsearch/pull/34733